src/MapUpMap.js

```diff
@@ -1,21 +1,40 @@
 import React from 'react';
 import Map from 'react-map-gl';
-
+import Footer from './Footer';
```

Comment:
Is Footer being used here? If it's unused I'd remove it, it can also cause warnings when running builds. VSCode should show in light grey if not required